slave/download.py

```diff
@@ -286,9 +286,9 @@ def retrieveInfo(self):
     parser.add_option("-u", "--url", dest="url",
                       help="Specify a specific url to download.", default=None)
     parser.add_option("--repo", dest="repo",
-                      help="Specify a repo to download. Currently only mozilla-inbound supported.", default=None)
+                      help="Specify a repo to download. Currently supports: mozilla-inbound (cset?) | chrome", default=None)
```

Comment:
webkit is also supported. I think we should remove the (cset?) and add in the line under it that anything else than latest is only support in mozilla-inbound. I'll update that when pushing.